cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -321,6 +324,34 @@ func gitIndex(c gitIndexConfig, o *indexArgs, l sglog.Logger) error {
 		"-submodules=false",
 	}
 
+	// We store the document ranks as JSON in gitDir and add a flag to tell
+	// zoekt-git-index to rank based on the scores in the file, instead of ranking
+	// the files by itself.
+	if o.UseOfflineRanking {
+		args = append(args, "-offline_ranking")
+
+		r, err := sourcegraph.GetDocumentRanks(context.Background(), o.Name)
+		if err != nil {
+			return err
```

Comment:
can you wrap these errors, I worry by an opaque error like "network failed" and then we don't know the source.